src/components/common/ColonyActions/helpers/getEventTitleValues.ts

```diff
@@ -171,4 +182,20 @@ const getEventTitleValues = (
   });
 };
 
-export default getEventTitleValues;
+/* Returns the correct message values for Motions according to the event type. */
+export const getMotionEventTitleValues = (
+  eventName: ColonyAndExtensionsEvents | SystemMessages,
+  actionData: ColonyAction,
+  motionMessageData?: MotionMessage,
+  motionMessageInitiatorUser?: User | null,
+) => {
```

Comment:
See my other comments but I think we can get away with just passing `eventName` and `motionMessageData` into this function